### PR TITLE
update UI documentation

### PR DESCRIPTION
**Pull request by sloanej**
This PR includes the updated UI templates in "Documentation/UI Template.fig". New screen templates are added for the profile screen and discussion-related screens. Additionally, the background images and icons used in the templates are stored in "EdxMobile/assets", and the icon paths are added to Icon.tsx.

### CODE REVIEW

File: Documentation/Assets/vector icons/angle-down-solid.svg
Comment (jonmahoney15) [inline Documentation/Assets/vector icons/angle-down-solid.svg:1]: all of the icons can be moved to the asset/icon folder in the EdxMobile instead of here

Comment (sloanej) [reply Documentation/Assets/vector icons/angle-down-solid.svg:1]: Ok. I moved them. The commit now reflects this change.

**Comment by jonmahoney15** [comment]
Okay this looks good, however the .DS_Store need to be removed, if you rebase your main branch, itll add the gitignore and should remove them

**Comment by sloanej** [comment]
> Okay this looks good, however the .DS_Store need to be removed, if you rebase your main branch, itll add the gitignore and should remove them

Okay.  I think we're good now.

**Review by jonmahoney15 — APPROVED** [review]

**Pull request merged**